pkg/types/powervs/platform.go

```diff
@@ -0,0 +1,63 @@
+package powervs
+
+// Platform stores all the global configuration that all machinesets
+// use.
+// Note: The subsequent mentions of future-TF support refer to work that is
+// undergoing and should be available to test well in time for 4.10 feature-
+// freeze. We do not plan to GA with these as required inputs.
+type Platform struct {
+
+	// ServiceInstanceID is the ID of the Power IAAS instance created from the IBM Cloud Catalog
+	ServiceInstanceID string `json:"serviceInstance"`
+
+	// PowerVSResourceGroup is the resource group for creating Power VS resources.
+	PowerVSResourceGroup string `json:"powervsResourceGroup"`
+
+	// Region specifies the IBM Cloud region where the cluster will be created.
+	Region string `json:"region"`
+
+	// Zone specifies the IBM Cloud colo region where the cluster will be created.
+	// Required for multi-zone regions.
+	Zone string `json:"zone"`
+
+	// Zone in the region used to create VPC resources. Leave unset
+	// to allow installer to randomly select a zone.
+	//
+	// +optional
+	VPCZone string `json:"vpcRegion,omitempty"`
+
+	// UserID is the login for the user's IBM Cloud account.
+	UserID string `json:"userID"`
+
+	// APIKey is the API key for the user's IBM Cloud account.
+	APIKey string `json:"APIKey,omitempty"`
+
+	// VPC is a VPC inside IBM Cloud. Needed in order to create VPC Load Balancers.
```

Comment:
The text here implies to me that you can have a successful install without specifying the VPC, in which case your cluster will not use VPC Load Balancers. Is that accurate? Or is the optional aspect of this like it is for other platforms where omitting the VPC tells the installer to create a VPC for you?

Reply:
ah -- i blanked on explaining that a few things here are temporary. we've had to pause on including the vpc creation, so for now there are manual steps for dev that we need to take to create a vpc and the networks in both ibmcloud and power vs. after that, vpc will be created by the installer. it's not going to be optional when we've got networking a fix into prod.

Reply:
Let's still clean up the text in the description to represent the current behavior.